server/datastore/mysql/apple_mdm.go

```diff
@@ -4753,127 +4752,6 @@ WHERE
 	return ctxerr.Wrap(ctx, err, "updating abm_token")
 }
 
-func (ds *Datastore) InsertVPPToken(ctx context.Context, tok *fleet.VPPTokenData, teamID *uint, nullTeam fleet.NullTeamType) (*fleet.VPPTokenDB, error) {
```

Comment:
I just moved those VPP-related datastore methods to the `vpp.go` file that we already had. It seems more natural a place to look for those.